compiler/luci-interpreter/src/kernels/Utils.h

```diff
@@ -31,6 +31,11 @@ namespace luci_interpreter
 namespace kernels
 {
 
+#define throwIfConditionFalse(a)                                                      \
```

Comment:
Please change the name to SCREAMING_SNAKE_CASE. It is our style for macro names.
`cond` or `condition` instead of vague `a` would also be better.

Reply:
Okay, I will fix it. and could you recommend the name of a function? If not, I will just use `THROW_IF_CONDITIONFALSE(cond)`.

Reply:
It's up to you :) Any name that clearly and unambiguously defines the purpose is OK to me.
Personally, I would use something like LUCI_INTERPRETER_CHECK.